includes/Admin/Assets.php

```diff
@@ -39,6 +39,11 @@ public function register_hooks(): void {
 		add_action( 'admin_enqueue_scripts', array( $this, 'enqueue_assets' ) );
 	}
 
+	public function enqueue_assets() {
```

Comment:
@asvinb the method is re-declared here. As per the webpack entry point, would it work if we replace `admin` with `index`?

Reply:
@Sidsector9 I see you removed the duplicate method. Let me know if that's ok now. Thanks!